index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <title>Spring 2024 ReDI Berlin JavaScript course</title>
+    <link rel="preconnect" href="https://unpkg.com" />
```

Comment:
sorry, couldnt help.
This will shave off that millisecond of loading time we desperately need :)

Reply:
It is the only domain we use though, this is foolproof

Reply:
maybe we should put it as the first element in the header, to shave off that most important nanosecond.